packages/dds/task-manager/src/taskManager.ts

```diff
@@ -192,7 +194,22 @@ export class TaskManager extends SharedObject<ITaskManagerEvents> implements ITa
     /**
      * Map to track tasks that have pending complete ops.
      */
-     private readonly pendingCompletedTasks: Map<string, number[]> = new Map();
+    private readonly pendingCompletedTasks: Map<string, number[]> = new Map();
+
+    /**
+     * Placeholder clientId for detached scenarios.
+     */
+    private readonly placeholderClientId: string = "placeholder";
```

Comment:
My preference is to have simple named constants in a non-member (e.g. `const placeholderClientId`) outside the class - basically anything that's trying to be a `#define` I prefer a non-member.